docs/source/guide/labeling.md

```diff
@@ -1,10 +1,9 @@
 ---
 title: Label and annotate data 
 type: guide
-tier: all
 order: 402
 meta_title: Label and annotate data
-meta_description: "Label and annotate data to create bounding boxes, label text spans, set up relations. Filter and sort project data for machine learning dataset creation."
+meta_description: Label and annotate data using the Label Studio UI to create bounding boxes, label text spans, set up relations, and filter and sort project data for your machine learning dataset creation.
```

Comment:
The original changes was a recommendation from Justin. Should we keep it?

Reply:
yes